server/src/main/java/org/candlepin/resource/GuestIdResource.java

```diff
@@ -157,38 +214,41 @@ public void updateGuest(
         @PathParam("consumer_uuid") @Verify(Consumer.class) String consumerUuid,
         @ApiParam("guest virtual uuid")
         @PathParam("guest_id") String guestId,
-        @ApiParam(name = "updated", required = true, value = "updated guest data to use") GuestId updated) {
+        @ApiParam(name = "updated", required = true, value = "updated guest data to use")
+        GuestIdDTO updatedDTO) {
```

Comment:
Don't need to wrap here

Reply:
we do.
resource/GuestIdResource.java:217: Line is longer than 110 characters (found 114). [LineLength]

Reply:
ack